check-code.js

```diff
@@ -0,0 +1,135 @@
+const shuffleRemainingItems = (array) => {
+    // Remove the first element from the array
+    array.shift();
+    // Shuffle the remaining elements in the array
+    // Iterate through the smaller array, working backwards to shuffle the items
+    for (let i = array.length - 1; i > 0; i--) {
+        // Create a variable 'j' that will be used to choose a random index from the subarray of unshuffled indices
+        // Initially, 'j' will be a random number from 0 to 3 for a smallerArray length of 4
+        let j = Math.floor(Math.random() * (i + 1));
+        // Create a temporary variable to store the value of the last element (index 'i') of the unshuffled subarray
+        let temp = array[i];
+        // Swap the value at the last element (index 'i') of the unshuffled subarray with a randomly chosen element in the unshuffled subarray (index 0 to index 'i')
+        // NOTE: It could be swapped with itself
+        array[i] = array[j]
+        // Store the initial unshuffled element at index 'i', now in temp, into index 'j'
+        array[j] = temp;
+    }
+    // Return shuffled array
+    return array;
+}
+
+const colors1 = ["purple", "blue", "green", "yellow", "pink"];
+const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"];
+
+console.log( shuffleRemainingItems(colors1) );
+console.log( shuffleRemainingItems(colors2) );
+
+
+// Create a function that takes in 1 argument that is an array
+const sumOfAllCubed = (array) => {
+    // Create a new array that stores the cube of the initial array's values
+    cubedArray = array.map(value => {
+        return value ** 3;
+    })
+
+    // Use an accumulator to go through the cubedArray to sum all the elements
+    let accumulator = 0;
```

Comment:
this would be a good opportunity to see if there is a built-in method that supports the logic you're building here